source/cloud/cloud-administration/sso-gitlab.rst

```diff
@@ -21,20 +34,31 @@ Step 1: Add an OAuth application to your GitLab account
       https://{mattermost-site-name}/signup/gitlab/complete
 
      If your GitLab instance is not set up to use SSL, your URIs must begin with ``http://`` instead of ``https://``.
+
   c. Do not select any options under **Scopes**.
 
-3. Click **Save application**.
+3. Select **Save application**.
 
-Keep the GitLab window open because you need the *Application Id* and *Secret* when you configure Mattermost.
+4. Keep the GitLab window open. You need the *Application Id* and *Applicatio Secret Key* when you configure Mattermost.
 
 Step 2: Configure Mattermost for GitLab SSO
--------------------------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Log in to Mattermost, then go to the **System Console > Authentication > OpenID Connect**.
+2. Select **GitLab** as the service provider.
+3. Copy and paste the *Application ID* from GitLab to Mattermost.
+4. Copy and paste the *Application Secret Key* from GitLab to Mattermost. 
+5. Enter the **Gitlab Site URL** of our GitLab instance. 
+
+  If your GitLab instance is not set up to use SSL, the endpoints must begin with ``http://`` instead of ``https://``. If you are using GitLab itself as your OAuth provider, use *gitlab.com*.
+
+6. Enter the **User API Endpoint**. 
+
+  Use ``https://{gitlab-site-name}/api/v3/user`` if you're running GitLab v8.17.8 or earlier.
+
+(Optional) Step 3: Forcing all users to sign up with SSO only
```

Comment:
Should this be "Force all users to sign up with SSO only" seems like the verb tense changes for this step.